lib/routes/readwise/list.ts

```diff
@@ -0,0 +1,145 @@
+            {
+                name: 'READWISE_ACCESS_TOKEN',
+                optional: false,
+                description: 'Visit `https://readwise.io/access_token` to get your access token.',
+            },
+        ],
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['read.readwise.io'],
+            target: '/list',
+        },
+    ],
+    name: 'Readwise Reader Document List',
+    maintainers: ['xbot'],
+    handler,
+    description: `Specify options (in the format of query string) in parameter \`routeParams\` to filter documents.
+
+| Parameter                  | Description                                                                            |   Values                                                                                    |  Default                             |
+| -------------------------- | -------------------------------------------------------------------------------------- |   ----------------------------------------------------------------------------------------- |  ----------------------------------- |
+| \`location\`               | The document's location.                                                               |   \`new\`/\`later\`/\`shortlist\`/\`archive\`/\`feed\`                                      |                                      |
+| \`category\`               | The document's category.                                                               |   \`article\`/\`email\`/\`rss\`/\`highlight\`/\`note\`/\`pdf\`/\`epub\`/\`tweet\`/\`video\` |                                      |
+| \`updatedAfter\`           | Fetch only documents updated after this date.                                          |   string (formatted as ISO 8601 date)                                                       ||
+| \`tag\`                    | The document's tag, can be specified once or multiple times.                           |||
+| \`tagStrategy\`            | If multiple tags are specified, should the documents match all of them or any of them. |   \`any\`/\`all\`                                                                           |  \`any\`                             |
+
+Customise parameter values to fetch specific documents, for example:
+
+\`\`\`
+https://rsshub.app/readwise/list/location=new&category=article
+\`\`\`
+
+fetches articles in the Inbox.
+
+\`\`\`
+https://rsshub.app/readwise/list/category=article&tag=shortlist&tag=AI&tagStrategy=all
+\`\`\`
+
+fetches articles tagged both by \`shortlist\` and \`AI\`. `,
+};
+
+const TAG_STRATEGY_ALL = 'all'; // Items with tags matching all the given ones can then be returned.
+const TAG_STRATEGY_ANY = 'any'; // Items with tags matching any of the given ones can be returned.
+
+async function handler(ctx) {
+    const accessToken = process.env.READWISE_ACCESS_TOKEN;
```

Comment:
Please set the token in [config.ts](https://github.com/DIYgod/RSSHub/blob/master/lib/config.ts) and throw an exception if not set.